### PR TITLE
Update docker/README.md for deploy.sh and template.

### PR DESCRIPTION
**Pull request by philbudne**
Also add deploy.sh check for empty string values.

### CODE REVIEW

**Comment by philbudne** [comment]
See last comment in https://github.com/mediacloud/story-indexer/issues/62

**Review by pgulley — APPROVED** [review]

File: docker/README.md [hunk 1]
```diff
@@ -8,36 +8,160 @@ Before you begin, ensure you have the following prerequisites in place:
...
+  -n      dry-run: creates docker-compose.yml but does not invoke docker (implies -a -u)
+  -u      allow running as non-root user
+```
+
+## docker-compose.yml.j2 template
+
+docker-compose.yml is generated the Jinja2 template
+docker-compose.yml.j2
+
+Read the comments in at the top of the template before opening a PR
```
Comment (kilemensi) [inline docker/README.md:140]:
```suggestion
docker-compose.yml is generated from the Jinja2 template
docker-compose.yml.j2

Read the comments at the top of the template before opening a PR
```

**Review by kilemensi — APPROVED** [review]

**Comment by philbudne** [comment]
@thepsalmist any comments/questions?

**Review by thepsalmist — APPROVED** [review]
LGTM

**Pull request merged**